### DISCUSSION TITLE
Missing appearances

### DISCUSSION THREAD

**Discussion by mortgad** [dcaribou/transfermarkt-datasets#271] [Q&A] [answered]
Hi David,

When I look at the Kaggle dataset it seems like the appearances.csv file is lacking values, since the date range goes from 2012-07-03 to 2023-06-11. I use them to calculate team values.

When I run my script now, I lack values since the end of the 2022/23 season, and I suspect it is due to the lack of appearances.

**Comment by dcaribou** [comment]
Hey @mortgad, can you provide some examples of such values that you found missing?

**Comment by mortgad** [comment]
Hi David,

If I go to your Kaggle dataset and view the appearances.csv file I can see the data range is 2012-07-03 - 2023-06-11.
Assuming I read that correctly, it means that the dataset currently does not hold appearances later than the 11th of June 2023.

**Reply by mortgad** [reply]
![image](https://github.com/dcaribou/transfermarkt-datasets/assets/48317690/d1a99f91-801f-41c4-94ae-93843343d432)

**Comment by dcaribou** [comment]
Thank you for the additional details @mortgad.
I've found the missing data leaked in this commit https://github.com/dcaribou/transfermarkt-datasets/commit/833bf3604ab92be5cdc937a1df3862d80114b39b from some days ago.
My guess is that this is due to a change in transfermarkt, but I'll need to investigate more.

In the meantime, I'll be restoring the missing data and blocking the updates until I find the root cause.

**Comment by dcaribou** [comment, marked as answer]
Update on this: the issue was indeed caused by a change in transfermarkt. It should be fixed by now. [This](https://github.com/dcaribou/transfermarkt-scraper/commit/34a2a3ee0d403e5e19596036fb205f4491ffc2fa) is the related fix in the scraper.

I'll try to add some tests, so errors like this are caught before they propagate to kaggle and other frontends in the future. Watch #195.

**Reply by mortgad** [reply]
Awesome, thank you! :)
